frontend/src/components/datasets/CreateLicenseModal.tsx

```diff
@@ -12,8 +12,9 @@ import {
 import licenseSchema from "../../schema/licenseSchema.json";
 import { FormProps } from "@rjsf/core";
 import { ClowderRjsfErrorList } from "../styledComponents/ClowderRjsfErrorList";
-import Form from "@rjsf/material-ui";
-import { V2 } from "../../openapi"; // Import LicenseOut type
+import Form from "@rjsf/mui";
+import { V2 } from "../../openapi";
+import validator from "@rjsf/validator-ajv8"; // Import LicenseOut type
```

Comment:
Maybe wrong inline comment "// Import LicenseOut type"?

Reply:
No idea how this was added. But will remove that.